sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -753,7 +753,7 @@ func (ihnc *indexHashedNodesCoordinator) computeNodesConfigFromList(
 	newNodesList := make([]Validator, 0)
 	auctionList := make([]Validator, 0)
 
-	if ihnc.flagWaitingListFix.IsSet() && previousEpochConfig == nil {
+	if previousEpochConfig == nil {
```

Comment:
if ihnc.flagStakingV4Started.IsSet() && previousEpochConfig == nil {   ?

Reply:
After discussions with @AdoAdoAdo , we will delete this "previousEpochConfig" completely in the next PR anyway, but I'll add it for consistency

Reply:
Ok